modules/mono/editor/GodotTools/GodotTools/Build/BuildOutputView.cs

```diff
@@ -95,12 +95,12 @@ private void LoadIssuesFromFile(string csvFile)
                         var issue = new BuildIssue
                         {
                             Warning = csvColumns[0] == "warning",
-                            File = csvColumns[1],
-                            Line = int.Parse(csvColumns[2]),
-                            Column = int.Parse(csvColumns[3]),
-                            Code = csvColumns[4],
-                            Message = csvColumns[5],
-                            ProjectFile = csvColumns[6]
+                            File = csvColumns[1] ?? "",
+                            Line = int.Parse(csvColumns[2] ?? ""),
+                            Column = int.Parse(csvColumns[3] ?? ""),
+                            Code = csvColumns[4] ?? "",
+                            Message = csvColumns[5] ?? "",
+                            ProjectFile = csvColumns[6] ?? ""
```

Comment:
This change shouldn't be needed either. `FileAccess::get_csv_line` will never return null strings.

```suggestion
```